app/routers/event.py

```diff
@@ -108,6 +117,15 @@ def by_id(db: Session, event_id: int) -> Event:
     return event
 
 
+def get_user_or_create_default(user: User, session: Session) -> User:
+    """Check if the user is logged in, else create a fictive user
```

Comment:
Add a blank line after the docstring's summary line :)

Reply:
I decided to remove the method, after it was edited by someone else.